packages/catalog-model/examples/apis/spotify-api.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     - rest
   annotations:
     # The annotation is deprecated, we use placeholders (see below) instead, remove it later.
```

Comment:
If we are making changes to this file we should probably deal with this comment and just remove this. Ideally we should just let the definition get pulled in on line 17 and make sure that URL is up to date

Reply:
yes, the updated URL is not used anymore actually. See `spec.definition.$text` for the actually used URL.